src/components/editor/InputListLOC.jsx

```diff
@@ -131,11 +126,12 @@ InputListLOC.propTypes = {
 }
 
 const mapStatetoProps = (state) => {
-  let data = state.lookups.formData
-  let result = {}
-  if (data !== undefined){
-    result = { formData: data }
-  }
+  // let data = state.lookups.formData
+  let result = Object.assign({}, state)
+  // if (data !== undefined){
+  //   result = { formData: data }
+  // }
```

Comment:
can commented out stuff go away?  or should it be mentioned in a ticket somewhere?  or ...

Reply:
This will part of the work  [#387](https://github.com/LD4P/sinopia_editor/issues/387), list item 3.